lib/yang_wrappers.c

```diff
@@ -1137,6 +1137,24 @@ void yang_str2mac(const char *value, struct ethaddr *mac)
     (void)prefix_str2mac(value, mac);
 }
 
+struct yang_data *yang_data_new_date_and_time(const char *xpath, time_t time)
+{
+	struct tm *tm;
+	char timebuf[MONOTIME_STRLEN];
+	struct timeval _time, time_real;
+
+	_time.tv_sec = time;
+	_time.tv_usec = 0;
+	monotime_to_realtime(&_time, &time_real);
+
+	tm = gmtime(&time_real.tv_sec);
```

Comment:
Need to use `gmtime_r`, other than that ACK

Reply:
changed to `gmtime_r`